### ISSUE TITLE
[Crash Android] App crashes through several actions

### ISSUE THREAD

**Issue by Switch123456789** [demokratie-live/democracy-client#1587]
The App crashes whenever I tab an entry in "Bundestag > Vorbereitung" & "Profile > Abgeordnetenprofil". I am a new user and I installed it through f-droid.

Fairphone 3 | iodeOS 3.6 | Android 12 | rooted

**Comment by madswillem** [comment]
I had the same Problem. The solution is to download the newest version from the website because the F-Droid version is 1.2.2 and the newest is 1.5.4.

 https://www.democracy-deutschland.de/

**Comment by Switch123456789** [comment]
I just noticed that shortly after reporting. I saw your comment at #1567 as well, seems like there quite some people who got caught off guard by this. On the [F-Droid Gitlab](https://gitlab.com/fdroid/fdroiddata/-/issues/2037) are some discussions going on about updating the F-Droid version but seemingly without a successful build for the past few years.

If there is no up-to-date build in sight, maybe there is something that can be done to prevent people from downloading this outdated version, maybe removing it from the store or flagging it as outdated or something.
